server-common/src/main/java/com/datastrato/gravitino/server/auth/KerberosServerUtils.java

```diff
@@ -1,13 +1,13 @@
-/**
+/*
  * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
  * agreements. See the NOTICE file distributed with this work for additional information regarding
  * copyright ownership. The ASF licenses this file to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance with the License. You may obtain a
  * copy of the License at
  *
- * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *   http://www.apache.org/licenses/LICENSE-2.0
  *
- * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * Unless required by applicable law or agreed to in writing, software distributed under the
```

Comment:
In general, you should never change license headers on a file without the owners's permission, but given the changes only impact formatting, that's probably fine.

Reply:
Thank you for bringing this to my attention. I will keep it in mind.